tv.kodi.Kodi.yml

```diff
@@ -115,11 +115,19 @@ modules:
     sources:
       - type: git
         url: https://github.com/xbmc/xbmc.git
-        commit: 918ff46a659fc12502e42cd7db62eaf250e07493
+        commit: e47331c6308b7f7f9760c8813c4f70fc150d4df1
       - type: patch
         path: kodi.sh.in.patch
-      - type: patch
-        path: addon-manifest.xml.patch
+      - type: file
+        path: addon-list.txt
+        dest: build/download/
+        dest-filename: addon-list.txt
```

Comment:
I think this can be omited, if the name matches? https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html

Reply:
probably, but is it worth triggering a full rebuild?

Reply:
Nope, but we could push it and cancel the build or include it in something that we're pushing later.

Reply:
sure. Let's update it with the next change.